src/cryptography/x509/verification.py

```diff
@@ -0,0 +1,9 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from cryptography.hazmat.bindings._rust import x509 as rust_x509
+
+Store = rust_x509.Store
+
+__all__ = ["Store"]
```

Comment:
Nit: put this between imports and declarations is our usual style

Reply:
152a12a26..293293506

Reply:
What can I say, we're hypocrites